src/views/ResultsPage/ResultsPage.jsx

```diff
@@ -474,7 +474,11 @@ const ResultsPage = () => {
 										Print Selected
 									</button>
 								</div>
-								<div className="results-page__pager">
+								<div
+									className={
+										'results-page__' +
+										(pagerExists ? 'pager' : 'no_pages_pager')
```

Comment:
see comment below.  these shouldnt be separate elements.  rather, it should be a modifier